src/github/queries.gql

```diff
@@ -270,6 +290,39 @@ query PullRequest($owner: String!, $name: String!, $number: Int!) {
 	}
 }
 
+query Issue($owner: String!, $name: String!, $number: Int!) {
+	repository(owner: $owner, name: $name) {
+		pullRequest: issue(number: $number) {

```

Comment:
I'm not sure if this is any faster, but it looks like there's a direct connection from `repository` to `issues`, so `pullRequest:` could be removed here and above

Reply:
I _thought_ I was just renaming the property by doing this so that I could re-use more code. Is that not the case?

Reply:
Ah, I see, you're able to use the `PullRequestResponseType` this way! I think that's fine for now. When doing the follow up PR to split out an `IssueModel`, I think it would be good to change this to query issues instead of doing the rename, and have a separate type for it.

Reply:
👍